README.md

```diff
@@ -42,6 +42,12 @@ This HMCC HK's Web Repo built on SailsJS and React
 3. When all code changes have been finished in your `feature` branch, create a Pull Request and your peers review it before merging.
 4. Branch merge flow should be: `feature`->`release`->`master` - i.e `feature` branches should always Pull Request and merge into `release`
 
+## Commit Practices
```

Comment:
lol how come you add this to the readme, but none of your commit messages follow this?

Reply:
hmm i also donno what happened